cmd/manager/main.go

```diff
@@ -86,6 +88,12 @@ func main() {
 	keplerImage := env("KEPLER_IMAGE", exporter.StableImage)
 	flag.StringVar(&exporter.Config.Image, "kepler.image", keplerImage, "kepler image")
 
+	keplerEstimatorImage := env("KEPLER_ESTIMATOR_IMAGE", estimator.StableImage)
+	flag.StringVar(&estimator.Config.Image, "kepler-estimator.image", keplerEstimatorImage, "kepler estimator image")
+
+	keplerModelServerImage := env("KEPLER_MODEL_SERVER_IMAGE", modelserver.StableImage)
+	flag.StringVar(&modelserver.Config.Image, "kepler-model-server.image", keplerModelServerImage, "kepler model server image")
```

Comment:
(suggestion): we have an opportunity to refactor this pattern of getting the env from the flag.
Lets use `estimator.image` and `model-server.image` in the flag please. 

I have a question about the name `model-server`. Do you know why we call it that? what does it really serve?

Reply:
model-server serves the model selection based on available features. When online-trainer is validated, it will serve the fine-tuned model online to the exporter.